MapleServer2/Types/StatDistribution.cs

```diff
@@ -20,19 +21,46 @@ public StatDistribution()
         {
             // hardcode the amount of stat points the character starts with temporarily
             this.TotalStatPoints = 18;
+            this.OtherStats = new Dictionary<byte, int>();
             this.AllocatedStats = new Dictionary<byte, int>();
+            AddTotalStatPoints(1, 0x2);
+            AddTotalStatPoints(2, 0x1);
+            AddTotalStatPoints(3, 0x3);
+            AddTotalStatPoints(4, 0x4);
         }
 
         public StatDistribution(Dictionary<byte, int> AllocatedStats)
         {
             this.AllocatedStats = AllocatedStats;
+            this.OtherStats = new Dictionary<byte, int>();
+        }
+
+        public StatDistribution(Dictionary<byte, int> AllocatedStats, Dictionary<byte, int> OtherStats)
```

Comment:
You can combine these two constructors by making `OtherStats` optional and having it initialize to a blank dictionary if none is passed in